template/jinja/macros.j2

```diff
@@ -0,0 +1,14 @@
+# -*- coding: utf-8 -*-
+# vim: ft=jinja
+#
+# Collection of common macros
+
+{%- macro format_kwargs(kwarg) -%}
```

Comment:
Need to be cautious with Jinja whitespace control `{%- ... -%}`: https://github.com/saltstack-formulas/collectd-formula/pull/141#issuecomment-485043263.